README.md

```diff
@@ -6,8 +6,8 @@
 | :--:     | :---:               | :---:        | :---:          | :---:            | :---:             | :---:                     |
 | [20.x][] | **Maintenance**     | [Iron][]     | 2023-04-18     | 2023-10-24       | 2024-10-22        | 2026-04-30                |
 | [22.x][] | **LTS**             | [Jod][]      | 2024-04-24     | 2024-10-29       | 2025-10-21        | 2027-04-30                |
-| [23.x][] | **Current**         |              | 2024-10-15     | -                | 2025-04-01        | 2025-06-01                |
-| 24.x     | **Pending**         |              | 2025-05-06     | 2025-10-28       | 2026-10-20        | 2028-04-30                |
+| [23.x][] | **Maintenance**     |              | 2024-10-15     | -                | 2025-04-01        | 2025-06-01                |
+| 24.x     | **Current**         |              | 2025-05-06     | 2025-10-28       | 2026-10-20        | 2028-04-30                |
```

Comment:
```suggestion
| [24.x][] | **Current**         |              | 2025-05-06     | 2025-10-28       | 2026-10-20        | 2028-04-30                |
```

Can you also prepare the link to `https://nodejs.org/download/release/latest-v24.x/` ?